### PR TITLE
93 debug view

### PR DESCRIPTION
**Pull request by hhirsch**

### CODE REVIEW

File: source/renderer/GameRenderer.cpp [hunk 1]
```diff
@@ -90,11 +90,17 @@ void GameRenderer::renderScene(RenderContext &renderContext) {
   renderEntities(renderContext);
 
   glClear(GL_DEPTH_BUFFER_BIT);
+  if(world.getConfig().isDebugViewEnabled()) {
```
Comment (SGOrava) [inline source/renderer/GameRenderer.cpp]: missing space after if
`if (world...`

**Comment by SGOrava** [comment]
i think it is better to use "enable" and no "enabled"

**Comment by hhirsch** [comment]
@SGOrava The "enable", "enabled" is a good thing to bring up. When I added my option there I saw "enabled" being used there before and when Dorian added his config settings he used "enable".

To me it does not make a big difference.
@SGOrava @ElementW can you each tell me your preference. I will open a new issue for it since it is unrelated to this issue and then use whatever we come up as the preference.

@SGOrava I fixed the format of the if clause. Thank you for pointing it out.

**Comment by ElementW** [comment]
@hhirsch IMO all options that are verbs should be in the infinitive form.

**Comment by hhirsch** [comment]
@ElementW Then infinitive it shall be. Putting this change in this same PR.

**Comment by hhirsch** [comment]
@SGOrava @ElementW Fixed.

**Pull request merged**